full-node/startup.sh

```diff
@@ -40,23 +39,22 @@ if [ "$EXEC_MODE" = "genesis" ]; then
     $BINARY_NAME init $MONIKER --chain-id $CHAIN_ID 2> /dev/null
     # staking/governance token is hardcoded in config, change this
     sed -i "s/\"stake\"/\"u${STAKE_DENOM}\"/" $CONFIG_DIR/genesis.json
-    sed -i "s/\"signed_blocks_window\": \".*$/\"signed_blocks_window\": \"${SLASHING_SIGNED_BLOCKS_WINDOW}\",/" $CONFIG_DIR/genesis.json
-    sed -i "s/\"downtime_jail_duration\": \".*$/\"downtime_jail_duration\": \"${DOWNTIME_JAIL_DURATION}\",/" $CONFIG_DIR/genesis.json
-    sed -i "s/\"min_signed_per_window\": \".*$/\"min_signed_per_window\": \"${SLASHING_MIN_SIGNED_PER_WINDOW}\",/" $CONFIG_DIR/genesis.json
-    sed -i 's/minimum-gas-prices = ""/minimum-gas-prices = "0.025u'"${DENOM}"'"/' $CONFIG_DIR/app.toml
+    sed -i "s/\"max_gas\": \".*$/\"max_gas\": \"${MAX_GAS}\",/" $CONFIG_DIR/genesis.json
+    sed -i "s/\"max_validators\": .*$/\"max_validators\": ${MAX_VALIDATORS},/" $CONFIG_DIR/genesis.json
+    sed -i "s/\"unbonding_time\": \".*$/\"unbonding_time\": \"${UNBONDING_TIME}s\",/" $CONFIG_DIR/genesis.json
+    sed -i 's/minimum-gas-prices = ""/minimum-gas-prices = "0.05u'"${DENOM}"'"/' $CONFIG_DIR/app.toml
```

Comment:
Can we change the `DENOM` and `STAKE_DENOM` value from `nois` to `unois`? Using `nois` is very confusing because at chain level unois is the base unit used everywhere.